README.md

```diff
@@ -2,12 +2,29 @@
 
 Multi-table operations over the lakehouse.
 
+![Multi-table operations](docs/images/DiscoverX_Multi-table_operations.png)
+
 Run a single command to execute operations across many tables. 
-Eg.
 
-* [Maintenance operations](#maintenance-operations) (vacuum, optimize, cleanup, etc.)
-* [Scan & classify](#scan--classify) content for patterns (phone numbers, emails, etc.)
-* Data transformations based on semantic types
+## Operations examples
```

Comment:
Is the following list ordered somehow w.r.t. importance or relevance. I was just thinking about semantic classification, which in the beginning was our main target and we list it almost in the end. Any thought behind this?

Reply:
I ordered them by how often I think they can be used.
Semantic classification is very powerful, but maintenance operations is something that everyone can/should do.